### PR TITLE
channel type test improvements

### PR DESCRIPTION
**Pull request by rustyrussell**

### CODE REVIEW

File: tests/test_bolt2-30-channel_type-open-accept-tlvs.py [hunk 1]
```diff
@@ -65,12 +60,66 @@ def test_open_channel(runner: Runner, with_proposal: Any) -> None:
...
+                channel_flags=1,
+                # We negotiate *down* to a simple non-static channel
+                tlvs='{channel_type={type=}}'),
+
+            # BOLT #2
+            #   - if it sets `channel_type`:
+            #     - MUST set it to the `channel_type` from `open_channel`
+            ExpectMsg('accept_channel',
+                      tlvs='{channel_type={type=}}')]
+
```
Comment (vincenzopalazzo) [inline tests/test_bolt2-30-channel_type-open-accept-tlvs.py:121]: here there this a missing `runner.run(test)`? this is the only reason to have the integration testing happy. The docker image is based on the actual c-lightning release.

**Review by vincenzopalazzo — CHANGES_REQUESTED** [review]
LGTM, but I have only a small NIT.

File: tests/test_bolt2-30-channel_type-open-accept-tlvs.py (hunk 1 above)
Comment (vincenzopalazzo) [reply tests/test_bolt2-30-channel_type-open-accept-tlvs.py:121]: ps: Ship make the magic here :)

**Comment by vincenzopalazzo** [comment]
ack https://github.com/rustyrussell/lnprototest/pull/28/commits/cdbf0fd5e94bda1d075a5e8c69fe96894b2d12f1

I fixed the python code formatting here and adding the runner code at the end of the test.

@rustyrussell Do you want this feature on c-lightning master? I can open a PR

**Review by vincenzopalazzo — APPROVED** [review]
ack https://github.com/rustyrussell/lnprototest/pull/28/commits/cdbf0fd5e94bda1d075a5e8c69fe96894b2d12f1

I fixed the python code formatting here and adding the runner code at the end of the test.

@rustyrussell Do you want this feature on c-lightning master? I can open a PR

**Comment by rustyrussell** [comment]
Good catch!  Fortunately it still passes :)

**Pull request merged**